packages/building-rollup/src/createSpaConfig.js

```diff
@@ -3,9 +3,10 @@
 /** @typedef {import('polyfills-loader').PolyfillsLoaderConfig} PolyfillsLoaderConfig */
 
 const merge = require('deepmerge');
-const html = require('@open-wc/rollup-plugin-html');
-const polyfillsLoader = require('@open-wc/rollup-plugin-polyfills-loader');
+const html = require('@web/rollup-plugin-html');
```

Comment:
Checking the failed tests:

https://github.com/open-wc/open-wc/pull/1986/checks?check_run_id=1407302256#step:7:309

The new `html` rollup plugin it's different?

Reply:
@LarsDenBakker Could you guide me a bit? Thanks in advance!

Reply:
I think this needs to be `const { rollupPluginHTML } = require('@web/rollup-plugin-html');`

Reply:
Updated! But same errors.

Reply:
@LarsDenBakker any other idea? Thanks!

Reply:
`const { polyfillsLoader } = require('@web/rollup-plugin-polyfills-loader');`

Reply:
@stephenwade done!

Reply:
@abdonrd I took another crack at this. I don't understand the remaining failing tests, but these changes should get you a step closer:

```diff
--- a/packages/building-rollup/src/createSpaConfig.js
+++ b/packages/building-rollup/src/createSpaConfig.js
@@ -49,8 +49,8 @@ function createSpaConfig(options) {

   const htmlPlugin = pluginWithOptions(rollupPluginHTML, userOptions.html, {
     transformHtml: [
-      [!userOptions.developmentMode && minifyHtml].filter(isFalsy),
-      [userOptions.injectServiceWorker && applySw].filter(isFalsy),
+      ...[!userOptions.developmentMode && minifyHtml].filter(isFalsy),
+      ...[userOptions.injectServiceWorker && applySw].filter(isFalsy),
     ],
   });

@@ -65,8 +65,8 @@ function createSpaConfig(options) {
     }
     outputDir = basicConfig.output[0].dir;

-    basicConfig.output[0].plugins.push(htmlPlugin.addOutput('module'));
-    basicConfig.output[1].plugins.push(htmlPlugin.addOutput('nomodule'));
+    basicConfig.output[0].plugins.push(htmlPlugin.api.addOutput('module'));
+    basicConfig.output[1].plugins.push(htmlPlugin.api.addOutput('nomodule'));
     polyfillsLoaderConfig = {
       modernOutput: {
         name: 'module',

Reply:
@LarsDenBakker Could you take a look at this? Not sure if the remaining failures mean that test cases need to be updated or if the plugin is configured wrong.